<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Video/Playlist URL:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File type / quality: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditURL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownloadLocation">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/folder.png</normaloff>img/folder.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownload">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDownloadLocation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelThumbnail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Thumbnail&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideoTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Video Title&gt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Location:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxQuality">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAudio">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxQuality">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAudio">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
